README.md

```diff
@@ -130,9 +130,19 @@ The following variables can store custom options that will be passed to fzf by t
 They are appended last to fzf's options list. Because fzf uses the last instance of an option if it is specified multiple times, custom options will always take precedence. Custom fzf options unlock a variety of customizations and augmentations such as:
 
 - add [key bindings](https://www.mankier.com/1/fzf#Key/Event_Bindings) within fzf to operate on the selected line:
-  - [open file in Vim](https://github.com/junegunn/fzf/issues/1360)
+
+  - [open file in in your `$EDITOR`](https://github.com/junegunn/fzf/issues/1360):
+
+    ```fish
+      # set -gx $EDITOR "nvim" # or "vim", or "code", etc.
+
+      # Ctrl-o will open the selected file/directory in your editor of choice.
+      set fzf_dir_opts --bind "ctrl-o:execute($EDITOR {} &> /dev/tty)"
```

Comment:
in case anyone is referencing this in the future, `fzf_dir_opts` has been renamed to `fzf_directory_opts`.